dojo/tools/jfrogxray/parser.py

```diff
@@ -0,0 +1,113 @@
+                    title_cve
+                items[unique_key] = item
+
+        return list(items.values())
+
+
+def get_item(vulnerability, test):
+    print(vulnerability)
+    # Following the CVSS Scoring per https://nvd.nist.gov/vuln-metrics/cvss
+    if 'severity' in vulnerability:
+        if vulnerability['severity'] == 'Unknown':
+            severity = "Info"
+        else:
+            severity = vulnerability['severity'].title()
+    # TODO: Needs UNKNOWN new status in the model.
+    else:
+        severity = "Info"
+
+    cve = "No CVE on file"
+    cwe = 0
+    cvss_v3 = "No CVSS v3 score."
+    cvss_v2 = "No CVSS v2 score."
+    mitigation = "N/A"
+    extra_desc = ""
+    # Some entries have no CVE entries, despite they exist. Example CVE-2017-1000502.
+    if 'cves' in vulnerability['component_versions']['more_details']:
+        if 'cve' in vulnerability['component_versions']['more_details']['cves'][0]:
+            cve = vulnerability['component_versions']['more_details']['cves'][0]['cve']
+
+        if 'cwe' in vulnerability['component_versions']['more_details']['cves'][0]:
+            # take only the first one for now, limitation of DD model.
+            cwe = vulnerability['component_versions']['more_details']['cves'][0]['cwe'][0].split('-')[1]
+
+            # some can be "NVD-CWE-noinfo"
+            if not type(cwe) is int:
+                cwe = 0
+
+        if 'cvss_v3' in vulnerability['component_versions']['more_details']['cves'][0]:
+            cvss_v3 = vulnerability['component_versions']['more_details']['cves'][0]['cvss_v3']
+
+        if 'cvss_v2' in vulnerability['component_versions']['more_details']['cves'][0]:
+            cvss_v2 = vulnerability['component_versions']['more_details']['cves'][0]['cvss_v2']
+
+    if 'fixed_versions' in vulnerability['component_versions']:
+        mitigation = "Versions containing a fix:\n"
+        mitigation = mitigation + "</br>".join(vulnerability['component_versions']['fixed_versions'])
+
+    if 'vulnerable_versions' in vulnerability['component_versions']:
+        extra_desc = "Versions that are vulnerable:\n\n"
+        extra_desc = extra_desc + "</br>".join(vulnerability['component_versions']['vulnerable_versions'])
+
+    # The 'id' field is empty? (at least in my sample file)
+    if vulnerability['id']:
+        title = vulnerability['id'] + " - " + str(cve) + " - " + vulnerability['component']
+    else:
+        title = str(cve) + " - " + vulnerability['component']
+
+    # create the finding object
+    finding = Finding(
+        title=title,
+        cve=cve,
+        cwe=cwe,
+        test=test,
+        severity=severity,
+        description=vulnerability['summary'] + "\n\n" + extra_desc,
+        mitigation=mitigation,
+        file_path=vulnerability.get('source_comp_id'),
+        severity_justification="CVSS v3 base score: {}\nCVSS v2 base score: {}".format(cvss_v3, cvss_v2),
```

Comment:
```suggestion
        severity_justification=f"CVSS v3 base score: {cvss_v3}\nCVSS v2 base score: {cvss_v2}",
```

Less verbose format string; [see docs here](https://docs.python.org/3/reference/lexical_analysis.html#f-strings). New in Python 3.6.